packages/cli/src/cli.ts

```diff
@@ -148,8 +148,8 @@ export interface ExtendedRoutesConfig extends RoutesConfig {
 }
 
 const validateRoutesConfig = async (config: Config): Promise<ExtendedRoutesConfig> => {
-  if (!config.entryFile) {
-    throw new Error('Missing entryFile: Configuration must contain an entry point file.');
+  if (!config.entryFile && !config.controllerPathGlobs) {
```

Comment:
Need to apply similar changes here: https://github.com/lukeautry/tsoa/blob/99502d0979e439b0b8ccfc6a58eca655022c8109/packages/cli/src/cli.ts#L94